package.json

```diff
@@ -20,6 +20,11 @@
     "storybook": "start-storybook -p 9009 -s public",
     "storybook:build": "build-storybook -s public"
   },
+  "husky": {
+    "hooks": {
+      "pre-commit": "npm run lint"
```

Comment:
https://github.com/okonet/lint-staged can be really helpful to improve developer experience (via running `lint-staged` on `pre-commit` to only check against staged files & `lint` on `pre-push` to ensure ESLint is passing for the whole)

Reply:
This would be a great addition to this check since the current eslint also takes a bit of time to finish. I will create an issue for that later.